File: Golang/IntegerToRoman/IntegerToRoman_test.go
package IntegerToRoman

import "testing"

func TestIntToRoman(t *testing.T) {

	if intToRoman(1) == "I" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if intToRoman(621) == "DCXXI" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if intToRoman(1996) == "MCMXCVI" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

	if intToRoman(999) == "CMXCIX" {
		t.Log("pass")
	} else {
		t.Error("failed")
	}

}

File: Python/IntegerToRomanTest.py
from unittest import TestCase

from IntegerToRoman import IntegerToRoman


class TestIntegerToRoman(TestCase):
    def test_intToRoman(self):
        i = IntegerToRoman()

        self.assertEqual(i.intToRoman(1), "I")

        self.assertEqual(i.intToRoman(621), "DCXXI")

        self.assertEqual(i.intToRoman(1996), "MCMXCVI")

        self.assertEqual(i.intToRoman(999), "CMXCIX")
